pkg/operator/condition/condition.go

```diff
@@ -0,0 +1,56 @@
+package condition
+
+const (
+	// ManagementStateDegradedConditionType represents a condition when the operator management state is changed from Managed to other value.
```

Comment:
is tue when ...

Reply:
Would not formulate it in a differential way. Better: is true if the ManagementState is not "Managed".